atproto/syntax/aturi.go

```diff
@@ -103,7 +103,7 @@ func (n ATURI) Normalize() ATURI {
 }
 
 func (a ATURI) MarshalText() ([]byte, error) {
-	return []byte(a.String()), nil
+	return []byte(string(a)), nil
```

Comment:
what is the motivation for this diff? does the compiler not optimize this already?

Reply:
I was getting compiler errors from it for some reason?

Reply:
![CleanShot 2023-08-24 at 17 06 26](https://github.com/bluesky-social/indigo/assets/1617325/316e5202-e8e2-4ce7-9bb9-e0829d4ac66a)

Reply:
Oh, yeah looks like there isn't a `String()` method on the `ATURI` type rn.